source/api/cypress-api/screenshot-api.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Cypress.Screenshot
+comments: false
+---
+
+The Screenshot API allows you set defaults for how screenshots are captured during {% url "`.screenshot`" screenshot %} and automatic screenshots taken during test failures. You can set defaults for the following:
+
+- Which parts of the screen to capture.
+- Whether to scale your application under test in the screenshot.
+- Whether to disable JavaScript timers and CSS animations when taking the screenshot.
+- Whether to automatically take screenshots when there are run failures.
+- Which, if any, elements to black out when taking the screenshot.
+- Whether to wait for the Command Log to synchronize before taking the screenshot.
+
+# Syntax
+
+```javascript
+Cypress.Screenshot.defaults(options)
+```
+
+## Arguments
+
+**{% fa fa-angle-right %} options**  ***(Object)***
+
+An object containing one or more of the following:
+
+Option | Default | Description
+--- | --- | ---
+`blackout` | `[]` | Array of string selectors used to match elements that should be blacked out when the screenshot is taken. Does not apply to `runner` captures.
+`capture` | `'fullPage'` | Which parts of the Test Runner to capture. This value is ignored for element screenshot captures. Valid values are `viewport`, `fullPage`, or `runner`. When `viewport`, your application under test is captured in the current viewport. When `fullPage`, your application under test is captured in its entirety from top to bottom. When `runner`, the entire browser viewport, including the Cypress Command Log, is captured.  For screenshots automatically taken on test failure, capture is always coerced to `runner`.
+`disableTimersAndAnimations` | `true`| When true, prevents JavaScript timers (`setTimeout`, `setInterval`, etc) and CSS animations from running while the screenshot is taken.
+`scale` | `false` | Whether to scale the app to fit into the browser viewport. This is always coerced to `true` for `runner` captures.
+`screenshotOnRunFailure` | `true` | When true, automatically takes a screenshot when there is a failure in Run mode.
+
+# Examples
+
+## Blackout elements before screenshot
+
+Elements that match the specified selectors will be blacked out from the screenshot, but only when the `capture` option is `viewport`. `blackout` is ignored is `capture` is `runner`.
+
+```javascript
+Cypress.SelectorPlayground.defaults({
```

Comment:
`Cypress.SelectorPlayground` -> `Cypress.Screenshot`

needs renaming all over this page